components/tekton-ci/base/repository.yaml

```diff
@@ -180,3 +180,10 @@ metadata:
   name: pipeline-service
 spec:
   url: "https://github.com/openshift-pipelines/pipeline-service"
+---
+apiVersion: pipelinesascode.tekton.dev/v1alpha1
+kind: Repository
+metadata:
+  name: multi-arch-controller
```

Comment:
Is this relevant?
https://github.com/redhat-appstudio/multi-arch-controller/blob/main/.tekton/multi-arch-controller-pull-request.yaml#L18

Reply:
Not yet, but if I have it here it gives me the option of setting up builds with automated PR creation in the same manner as other repositories.

Reply:
Building in RHTAP was ok for the initial PR, but if we start iterating rapidly on this it will likely be a lot more convenient to have the automation.